MMR.Randomizer/Resources/REQ_CASUAL.txt

```diff
@@ -2938,10 +2906,27 @@
       "Id": "ItemGreatBayCompass",
       "RequiredItems": [
         "AreaGreatBayTempleAccess",
-        "Exit Great Bay Temple",
-        "MaskZora"
+        "Exit Great Bay Temple"
+      ],
+      "ConditionalItems": [
+        [
+          "MaskZora"
+        ],
+        [
+          "GBT Cutscene Diving",
+          "GBT Goron Waterwheel",
+          "GBT Red Valve"
```

Comment:
Tried to figure this one out but couldn't. Can you explain?

Reply:
I think I don't understand the whole concept behind "GBT Red Valve"

Reply:
It's probably my fault for not renaming this entry because it should be renamed to "GBT Red Valve without Zora Mask". Doing GBT without Zora or FD using cutscene diving hard requires Soaring for anything other than the Ice Arrow chest or the Map Chest because you cannot leave the Map Chest room without a death warp, which is a trick I will probably add in the future. Basically, "GBT Red Valve" just ensures Goron has a method to get to the red valve in the water control room to change the water direction.

Reply:
Shouldn't it be `GBT Reverse Water Direction` in that case?

Reply:
The methods for doing GBT with and without Zora are just totally separately written, although recycling `GBT Reverse Water Direction` could be a more efficient way of doing it. `GBT Red Valve` does check if you need Soaring or not, where the former entry does not.